### PR TITLE
chore (v0 vr-tests): Remove usage of Screener

### PR DESCRIPTION
**Pull request by TristanWatanabe**
## Changes: 
- removes Screener related `.steps.` and `commonScreenerSteps` files from `@fluentui/docs` since they are no longer needed.
- removes all Screener related files from `scripts/screener`.
- removes Screener related files (`vr-build` and `vr-test`) from `scripts/gulp/tasks`.
- removes `vr-build` and `vr-test` from `gulpfile.ts`
- updates config files to remove mention of Screener.
- updates `screenerSteps` export to `StoryWrightSteps` for an existing v0 vr-test.
- not completely northstar related, but also removes Screener related just-task.

## Related Issue(s)

* Part of #25078

### CODE REVIEW

File: packages/fluentui/docs/.eslintrc.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "overrides": [
     {
       // These files don't have actual dependencies
-      "files": ["src/examples/**/*.steps.ts", "src/examples/**/commonScreenerSteps.ts"],
+      "files": ["src/vr-tests/**/*.stories.tsx", "src/vr-tests/**/*.stories.ts", "src/vr-tests/**/*.ts"],
```
Comment (TristanWatanabe) [inline packages/fluentui/docs/.eslintrc.json:6]: Will create a follow-up PR removing all the `eslint-disable-next-line import/no-extraneous-dependencies` lines that were added to stories within the `vr-tests` subfolder

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/cf10f5be18439d0c0c3f53baac7f996237569691'>cf10f5be18439d0c0c3f53baac7f996237569691</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/fluentui/docs/package.json [hunk 2]
```diff
@@ -73,8 +73,6 @@
     "screener:build": "yarn build:storybook",
```
Comment (TristanWatanabe) [inline packages/fluentui/docs/package.json:73]: `screener:build` will be renamed along with v8 and v9 in a separate PR

File: packages/fluentui/docs/src/app.tsx [hunk 3]
```diff
@@ -23,8 +23,7 @@ import Routes from './routes';
 // Experimental dev-time accessibility attributes integrity validation.
 import { setup } from '@fluentui/ability-attributes';
 
-// Temporarily disabling the validation for Screener.
-if (process.env.NODE_ENV !== 'production' && !process.env.SCREENER) {
+if (process.env.NODE_ENV !== 'production') {
```
Comment (TristanWatanabe) [inline packages/fluentui/docs/src/app.tsx:26]: Not totally sure what this does so didn't remove the entire check and only removed the Screener one.

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test-react-components/Avatar-mount_base.html">1288</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test-react-components/Avatar-mount.html">1258</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test-react-components/Button-mount_base.html">924</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test-react-components/Button-mount.html">906</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test-react-components/FluentProvider-mount_base.html">1492</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test-react-components/FluentProvider-mount.html">1506</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test-react-components/FluentProviderWithTheme-mount_base.html">569</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test-react-components/FluentProviderWithTheme-mount.html">565</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">539</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">542</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">583</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">580</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test-react-components/MakeStyles-mount_base.html">1962</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test-react-components/MakeStyles-mount.html">1960</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test-react-components/Persona-mount_base.html">2774</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test-react-components/Persona-mount.html">2787</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test-react-components/SpinButton-mount_base.html">2387</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test-react-components/SpinButton-mount.html">2367</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: packages/fluentui/docs/.eslintrc.json (hunk 1 above)
Comment (TristanWatanabe) [reply packages/fluentui/docs/.eslintrc.json:6]: #26054

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/BaseButton-mount_base.html">1378</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/BaseButton-mount.html">1396</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Breadcrumb-mount_base.html">3417</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Breadcrumb-mount.html">3521</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Checkbox-mount_base.html">3038</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Checkbox-mount.html">3042</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/CheckboxBase-mount_base.html">2718</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/CheckboxBase-mount.html">2699</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/ChoiceGroup-mount_base.html">5170</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/ChoiceGroup-mount.html">5083</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/ComboBox-mount_base.html">1434</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/ComboBox-mount.html">1396</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/CommandBar-mount_base.html">11226</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/CommandBar-mount.html">10984</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/ContextualMenu-mount_base.html">12361</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/ContextualMenu-mount.html">12556</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/DefaultButton-mount_base.html">1616</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/DefaultButton-mount.html">1592</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/DetailsRow-mount_base.html">4195</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/DetailsRow-mount.html">4193</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/DetailsRowFast-mount_base.html">4177</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/DetailsRowFast-mount.html">4198</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/DetailsRowNoStyles-mount_base.html">4043</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/DetailsRowNoStyles-mount.html">4023</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Dialog-mount_base.html">3567</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Dialog-mount.html">3569</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/DocumentCardTitle-mount_base.html">670</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/DocumentCardTitle-mount.html">693</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Dropdown-mount_base.html">3712</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Dropdown-mount.html">3773</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/FocusTrapZone-mount_base.html">2382</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/FocusTrapZone-mount.html">2312</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/FocusZone-mount_base.html">2304</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/FocusZone-mount.html">2295</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/GroupedList-mount_base.html">2188</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/GroupedList-mount.html">2418</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/GroupedList-virtual-rerender_base.html">1307</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/GroupedList-virtual-rerender.html">1303</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">1956</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/GroupedList-virtual-rerender-with-unmount.html">1861</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/GroupedListV2-mount_base.html">650</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/GroupedListV2-mount.html">662</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/GroupedListV2-virtual-rerender_base.html">636</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/GroupedListV2-virtual-rerender.html">641</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">650</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">648</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/IconButton-mount_base.html">2255</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/IconButton-mount.html">2234</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Label-mount_base.html">854</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Label-mount.html">854</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Layer-mount_base.html">4977</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Layer-mount.html">5074</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Link-mount_base.html">960</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Link-mount.html">949</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/MenuButton-mount_base.html">1923</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/MenuButton-mount.html">1935</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/MessageBar-mount_base.html">2726</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/MessageBar-mount.html">2756</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Nav-mount_base.html">3717</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Nav-mount.html">3778</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/OverflowSet-mount_base.html">1556</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/OverflowSet-mount.html">1570</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Panel-mount_base.html">3009</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Panel-mount.html">2964</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Persona-mount_base.html">1500</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Persona-mount.html">1523</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Pivot-mount_base.html">1865</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Pivot-mount.html">1918</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/PrimaryButton-mount_base.html">1751</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/PrimaryButton-mount.html">1762</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Rating-mount_base.html">8101</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Rating-mount.html">8168</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/SearchBox-mount_base.html">1762</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/SearchBox-mount.html">1773</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Shimmer-mount_base.html">3325</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Shimmer-mount.html">3311</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Slider-mount_base.html">2502</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Slider-mount.html">2450</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/SpinButton-mount_base.html">5449</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/SpinButton-mount.html">5497</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Spinner-mount_base.html">917</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Spinner-mount.html">939</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/SplitButton-mount_base.html">3577</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/SplitButton-mount.html">3528</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Stack-mount_base.html">956</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Stack-mount.html">944</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/StackWithIntrinsicChildren-mount_base.html">2712</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/StackWithIntrinsicChildren-mount.html">2649</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/StackWithTextChildren-mount_base.html">5596</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/StackWithTextChildren-mount.html">5565</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/SwatchColorPicker-mount_base.html">11953</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/SwatchColorPicker-mount.html">12544</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/TagPicker-mount_base.html">3050</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/TagPicker-mount.html">3049</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/TeachingBubble-mount_base.html">95451</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/TeachingBubble-mount.html">95988</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Text-mount_base.html">929</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Text-mount.html">913</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/TextField-mount_base.html">1863</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/TextField-mount.html">1823</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/ThemeProvider-mount_base.html">1764</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/ThemeProvider-mount.html">1772</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/ThemeProvider-virtual-rerender_base.html">1240</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/ThemeProvider-virtual-rerender.html">1247</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2498</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2499</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Toggle-mount_base.html">1269</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/Toggle-mount.html">1307</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/buttonNative-mount_base.html">621</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26053/perf-test/buttonNative-mount.html">631</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by fabricteam** [comment]
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26053&commitId=56022613b8483915f368e69d0ff38a50dfff4c70&env=prod)

✅ There was 4 screenshots added, 0 screenshots removed, 1041 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9424"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Comment by fabricteam** [comment]
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26053&commitId=56022613b8483915f368e69d0ff38a50dfff4c70&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Animation.Animation Example Fill Mode.chromium.png|58| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>

File: packages/fluentui/docs/src/app.tsx (hunk 3 above)
Comment (layershifter) [reply packages/fluentui/docs/src/app.tsx:26]: @TristanWatanabe this adds a panel with validation warnings. We don't want to have it to be visible in production builds anyway, so let's keep it.

**Review by layershifter — APPROVED** [review]

**Review by ling1726 — APPROVED** [review]

File: scripts/tsconfig.json [hunk 4]
```diff
@@ -50,9 +50,6 @@
     {
       "path": "./monorepo/tsconfig.json"
     },
-    {
-      "path": "./screener/tsconfig.json"
```
Comment (Hotell) [inline scripts/tsconfig.json:54]: 😻

**Pull request merged**